german/Alkohol/feuerzeug/feuerzeug.md

```diff
@@ -0,0 +1,8 @@
+Beim Absuchen des Bootes haben Sie ein Feuerzeug entdeckt. Sie klopen auf ihre Hosentaschen - leer.
+Ihr gängiger Lösungsansatz "erstmal eine Rauchen" führt Sie hier nicht weiter. 
+Außerdem läuft ihnen die Zeit davon, denn das Wasser sammelt sich immer weiter im Boot.
+
+Was nun?
+
+[Nocheinmal um mich sehen](../alkohol.md)
+[Bierflasche öffnen - Prost](Prost/Prost.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.